.travis.yml

```diff
@@ -1,32 +1,24 @@
 language: node_js
-node_js:
-    - "8"
 
-env:
-  - CXX=g++-4.8
+services:
+    - xvfb
+
+node_js:
+    - "10"
 
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
+env:
+  - CXX=g++-4.8
 
 install:
-    - npm install xvfb-maybe
     - npm install
     - npm install codecov -g
 
-cache:
```

Comment:
Could you not remove cache directories? This is useful for doing faster runs on dev, master, and long running PRs

Reply:
https://docs.travis-ci.com/user/caching/#npm-cache

maybe this should be
```yml
cache: npm
```

Reply:
Oops I missed this:

```
Please note that as of July 2019, npm is cached by default on Travis CI
```